### PR TITLE
199 exam result view api

### PR DESCRIPTION
**Pull request by anand-it**
Exam Result view Modificaton

Teacher
image
Admin
image
Student
image

### CODE REVIEW

File: server/database/teacher/query.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ interface ExamScheduleQueryInterface {
     updateExamSchedule: (data: ExamSchedular) => string;
     getAllExamSchedule: (schoolId: any, date: string) => Promise<any>;
     getExamScheduleByClassId: (schoolId: any, date: string, classid: string) => Promise<any>;
+    getExamSchedulesByClassId: (schoolId: any, examid: string, classid: string) => Promise<any>;
```
Comment (ArunprasadRajendran) [inline server/database/teacher/query.ts:31]: change method name to getExamScheduleByExamIdAndClassId

File: server/routes/teacher/examScheduleInfo.ts [hunk 2]
```diff
@@ -52,6 +52,24 @@ router.get('/examSchedule/student', async (req: Request, res: Response) => {
     }
 });
 
+// get examschedule by classid
+router.get('/examSchedule/class', async (req: Request, res: Response) => {
```
Comment (ArunprasadRajendran) [inline server/routes/teacher/examScheduleInfo.ts:56]: change '/examSchedule/class url' to '/examSchedule/exam/class'

**Review by ArunprasadRajendran — APPROVED** [review]

File: server/database/teacher/query.ts (hunk 1 above)
Comment (anand-it) [reply server/database/teacher/query.ts]: Done

File: server/routes/teacher/examScheduleInfo.ts (hunk 2 above)
Comment (anand-it) [reply server/routes/teacher/examScheduleInfo.ts]: done

**Pull request merged**